lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,31 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  end
+
+  def self.by_username(username)
+    user = User.find_by(username: username)
+    { user: user.public_user_attributes,
+      statistics: user_problem_statistcs(user) }
+  end
```

Comment:
Another style thing (that is inconsistent throughout the codebase, so don't worry about it if you don't want to).

I don't worry about extra lines, and prefer the symmetry of

```
{
  a: "aye",
  b: "bee",
}
```

over the asymmetry of

```
{ a: "aye",
  b: "bee" }
```